docs/planning/netstandard-vnext/README.md

```diff
@@ -0,0 +1,117 @@
+# .NET Standard vNext
+
+This document describes the plan for .NET Standard 2.1, which includes the
+definition of its API surface.
+
+## Goals
+
+* **Include `Span<T>`**. In .NET Core 2.1 we've added `Span<T>` which is an
+  array like type that allows representing managed and unmanaged memory in a
+  uniform way and supports cheap slicing. It's at the heart of most performance-
+  related improvements in .NET Core 2.1 as it allows managing buffers in a more
+  efficient way as it helps in reducing allocations and copying. `Span<T>` is
+  considered a primitive type and requires runtime and compiler support in order
+  to be fully leveraged. Please note that when refer to `Span<T>` in this
+  document, we really mean the family of span-related features which includes
+  other types such as `ReadOnlySpan<T>`, `Memory<T>`, `ReadOnlyMemory<T>`, and
+  the new `System.Buffers` namespace.
+
+* **Include core-APIs working with spans**. While `Span<T>` is available as a
+  .NET Standard compatible NuGet package (`System.Memory`) adding this package
+  cannot provide extend the members of .NET Standard types that deal with spans.
```

Comment:
Looks like an extra word here -> "provide extend". Should probably omit "provide".

Reply:
Good catch. Fixed!